charts/validator/crds/tor-controller.yaml

```diff
@@ -12873,7 +12873,7 @@ spec:
                 description: HealthProbeBindAddress is the TCP address that the controller should bind to for serving health probes
                 type: string
               livenessEndpointName:
-                description: LivenessEndpointName, defaults to "healthz"
+                description: LivenessEndpointName, defaults to "livez"
```

Comment:
Do not change this please. This a custom resource definition for the tor controller. Not related to suite code but rather the tor controller operator